apps/backend/db/migrations/20251212130537_merge-queue.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+export const up = async (knex) => {
+  await knex.schema.table("builds", (table) => {
+    table.boolean("mergeQueue").defaultTo(false).notNullable();
```

Comment:
Since PG 11 the table doesn't have to be rewritten to add a default value to a column so this like is good.